extensions/mathjax/renderer.go

```diff
@@ -13,17 +14,36 @@ import (
 //go:embed js
 var js embed.FS
 
-const script = `<script async src="/js/mathjax.js"></script>`
+const script = `
+<script>
+MathJax = {
+  tex: {
+    displayMath: [['$$', '$$'], ['\\[', '\\]']],
+    inlineMath: [['$', '$'], ['\\(', '\\)']]
+  },
+  svg: {
+    fontCache: 'global'
+  }
+};
+</script>
+<script type="text/javascript" id="MathJax-script" async
+  src="/js/tex-chtml-full.js">
+</script>
+    `
 
 func init() {
 	RegisterStaticDir(js)
-	RegisterBuildPage("/js/mathjax.js", false)
+	RegisterWidget(HEAD_WIDGET, 1, headScript)
```

Comment:
the problem with registering a widget here is that the script will be added to any page regardless of using mathjax. we must find a way to include the script only when mathjax is used in the page.